package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@stitch-fix/log-weasel",
+  "description": "Log Weasel Javascript client",
+  "version": "0.0.6",
+  "author": "Stitch Fix",
+  "license": "UNLICENSED",
+  "main": "index.js",
+  "repository": "https://github.com/stitchfix/log_weasel",
+  "files": [
+    "index.js",
+    "src/stitch_fix/logWeasel.js"
+  ],
+  "jest": {
+    "testURL": "http://localhost"
+  },
+  "scripts": {
+    "clean": "rm build/app.js",
+    "test": "jest",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "ulid": "^2.3.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.11.1",
+    "@babel/preset-env": "^7.11.0",
+    "@stitch-fix/browserslist-config-stitchfix": "^1.0.0",
```

Comment:
We'll also need to go with generic/open-source browserslist and eslint config eventually.

Reply:
There's not really such a thing as a generic browserslist (every app has different audiences and support strategies), aside from compiling down to ES5. This is so small, it would probably be fine to compile down to ES5, but if we wanted to give more control a note on importing the uncompiled source either through `module` or just by path would be good.